sccp/imports/scRNA.py

```diff
@@ -135,20 +163,6 @@ def load_lupus_data(third_axis="ind_cov", every_n=4):
     # keep only n observations
     X = X[::every_n, :]
 
-    # get cell types -> only stay true if order of cells doesn't change after this
```

Comment:
lines 138-151 that I get rid of here are essentially what I add to S1 and S2. that's all that happens